db/schema.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS usersData;
+
+CREATE TABLE usersData(
+   id        INTEGER  NOT NULL PRIMARY KEY 
+  ,age       INTEGER 
+  ,name      VARCHAR(15)
+  ,company   VARCHAR(9)
+  ,country   VARCHAR(11)
+  ,photo     text
+  ,about     VARCHAR(483)
```

Comment:
`about` should be a text field (not varchar) because we can't predict how long these paragraphs will be.